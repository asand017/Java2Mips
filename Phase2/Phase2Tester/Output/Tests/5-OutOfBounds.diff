--- ./Phase2Tester/SelfTestCases/5-OutOfBounds.out	2011-04-28 14:55:20.000000000 -0700
+++ ./Phase2Tester/Output/Tests/5-OutOfBounds.out	2017-05-21 21:54:31.270001547 -0700
@@ -1,3 +1,4 @@
-0
---- Error (Explicit) ---
-array index out of bounds
+--- Error (ProgramFault) ---
+expecting argument 1 to be an integer, found undefined value
+  Main() line 12
+  A.run(<Heap.0>+0) line 18
