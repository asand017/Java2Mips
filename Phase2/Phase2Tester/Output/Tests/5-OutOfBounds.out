--- Error (ProgramFault) ---
expecting argument 1 to be an integer, found undefined value
  Main() line 12
  A.run(<Heap.0>+0) line 18
